acceptance/tests/validate_vendored_ruby.rb

```diff
@@ -27,7 +27,7 @@ def setup_build_environment(agent)
   when /el-5/
     # PA-1638
     gem_install_sqlite3 += " -v 1.3.11"
-  when /solaris-11-i386/
+  when /solaris-11(.4|)-i386/
```

Comment:
Does that . need to be escaped?

Reply:
Only if we ever expect to have platform names that match  `/solaris-11.4/` where that character isn't a literal `.`. We probably should've caught that in the original PR, but since this is a cherry-pick I think we're safe to let it slide.

Reply:
No worries.